### ISSUE TITLE
Eslint error when opening a javascript file: Failed to load plugin jsx-a11y: Cannot find module 'eslint-plugin-jsx-a11y'

### ISSUE THREAD

**Issue by alfiepoleon** [Microsoft/vscode#10954]
- VSCode Version: 1.4.0 (1.4.0)
- OS Version: OS X El Capitan 10.11.3 (15D21)

I haven't been able to use eslint with my meteor project, it brings up this error everytime
`Failed to load plugin jsx-a11y: Cannot find module 'eslint-plugin-jsx-a11y'`
It works and lints fine with atom and sublime text, but I want to try out VS Code.
I have installed the ESLint extension.
This is my .eslintrc.json
```
{
    "parserOptions": {
        "ecmaVersion": 6,
        "esversion": 6,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "plugins": [
        "jsx-a11y",
        "meteor",
        "react"
    ],
    "extends": [
        "airbnb",
        "plugin:meteor/recommended"
    ],
    "env": {
        "meteor": true,
        "browser": true,
        "node": true,
        "es6": true,
        "mongo": true
    },
    "rules": {
        "meteor/no-session": [
            0
        ],
        "meteor/prefer-session-equals": [
            2
        ]
    },
    "globals": {
        "FlowRouter": false,
        "GoogleMaps": false,
        "moment": false
    }
}

```
And my package.json
```
{
  "name": "***",
  "private": true,
  "author": "***",
  "devDependencies": {
    "admin-lte": "^2.3.3",
    "eslint": "^2.11.1",
    "eslint-config-airbnb": "^9.0.1",
    "eslint-plugin-html": "^1.5.1",
    "eslint-plugin-import": "^1.8.1",
    "eslint-plugin-jsx-a11y": "^1.5.5",
    "eslint-plugin-meteor": "^3.5.2",
    "eslint-plugin-react": "^5.1.1"
  },
  "peerDependencies": {
    "eslint": "^2.11.1",
    "eslint-config-airbnb": "^9.0.1",
    "eslint-plugin-html": "^1.5.1",
    "eslint-plugin-import": "^1.8.1",
    "eslint-plugin-jsx-a11y": "^1.5.5",
    "eslint-plugin-meteor": "^3.5.2",
    "eslint-plugin-react": "^5.1.1"
  },
  "scripts": {
    "lint": "eslint .",
    "test": "npm run lint",
    "start": "meteor run"
  },
  "dependencies": {
    "babel-polyfill": "^6.9.1",
    "bcrypt": "^0.8.7",
    "country-data": "0.0.24",
    "jquery": "^1.9.1",
    "lodash.find": "^4.4.0",
    "lodash.memoize": "^4.1.0",
    "meteor-node-stubs": "~0.2.0",
    "paypal-ec": "^0.2.7",
    "paypal-rest-sdk": "https://registry.npmjs.org/paypal-rest-sdk/-/paypal-rest-sdk-1.6.9.tgz",
    "react": "^15.2.1",
    "react-dom": "^15.2.1",
    "sendgrid": "^3.0.9",
    "slideout": "^0.1.12",
    "tern": "^0.18.0"
  }
}
```

**Comment by sandy081** [comment]
This issue was moved to Microsoft/vscode-eslint#119

**Comment by Moejoe90** [comment]
Make sure you download all the dependancies and if this is a local installation then try installing all packages globally
